app.py

```diff
@@ -19,9 +19,14 @@ def login():
 def signup():
     return render_template('signup.html')
 
-@app.route('/lynn')
+@app.route('/lynn', methods=['GET', 'POST'])
 def lynn():
-    return render_template('lynn.html')
+    message = ""
```

Comment:
This is not necessary. We don't really need to declare an empty string. Try it out on your terminal :)